app/controllers/posts_controller.rb

```diff
@@ -6,6 +6,16 @@ def index
   end
 
   def show
+    page = MetaInspector.new(@post.url)
+    url_split = page.title.split(/: /)[0].split(/- /)
+    markdown = Redcarpet::Markdown.new(MdEmoji::Render, :no_intra_emphasis => true)
+
+    @url_img = page.images.best
+    @url_favicon = page.images.favicon
+    @url_title = url_split[0]
+    @url_subtitle = url_split[1]
+    @url_description = markdown.render(page.description).html_safe
+    @url_url = page.url
```

Comment:
Consider as a smell when having lots of instance variables in controller. It's better to wrap then as a PORO, so that in view you'll use (`object.url`, object.title`, ...)